server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/Main.scala

```diff
@@ -48,8 +53,7 @@ object Main extends IOApp {
       )
       metadataCache <- cachedResource[RelativePath, AdaptedGcsMetadataCache](
         storageAlgRef,
-        appConfig.stagingBucketName,
-        appConfig.metadataJsonBlobName,
+        appConfig.getSourceUri,
```

Comment:
this is wrong...the blob path for storagelink and metadata cache is different

Reply:
think I fixed it properly, with the assumption this is only in use for google currently...

Reply:
it'll be used for azure too..but I'll fix in my follow up PR